cypress/support/fragments/orders/orders.js

```diff
@@ -185,60 +179,43 @@ export default {
   },
 
   cancelOrder: () => {
-    cy.do([
-      orderDetailsPane
-        .find(PaneHeader({ id: 'paneHeaderorder-details' }).find(actionsButton))
-        .click(),
-      Button('Cancel').click(),
-      Button('Submit').click(),
-    ]);
+    expandActionsDropdown();
+    cy.do([Button('Cancel').click(), Button('Submit').click()]);
     InteractorsTools.checkCalloutMessage('Order was closed');
   },
 
   editOrderToManual: (orderNumber) => {
-    cy.do([
-      orderDetailsPane
-        .find(PaneHeader({ id: 'paneHeaderorder-details' }).find(actionsButton))
-        .click(),
-      Button('Edit').click(),
-      Checkbox({ name: 'manualPo' }).click(),
-      saveAndClose.click(),
-    ]);
+    expandActionsDropdown();
+    cy.do([Button('Edit').click(), Checkbox({ name: 'manualPo' }).click(), saveAndClose.click()]);
     InteractorsTools.checkCalloutMessage(
       `The Purchase order - ${orderNumber} has been successfully saved`,
     );
   },
 
-  unOpenOrder: () => {
-    cy.do([
-      orderDetailsPane
-        .find(PaneHeader({ id: 'paneHeaderorder-details' }).find(actionsButton))
-        .click(),
-      Button('Unopen').click(),
-      Button({ id: 'clickable-order-unopen-confirmation-confirm-delete-holdings' }).click(),
-    ]);
+  unOpenOrder({ orderNumber, confirm = true } = {}) {
```

Comment:
Why you do this? You add to method parameters, but in test files you delete parameters from method?